### PR TITLE
 switch to using new webconsole config

### PR DESCRIPTION
**Pull request by deads2k**
/assign spadgett

### CODE REVIEW

**Comment by spadgett** [comment]
Hm

> IOError: [Errno 2] No such file or directory: '/data/started.json'

/retest

File: pkg/apis/webconsole/util/helpers.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+	refs = append(refs, &config.ServingInfo.CertFile)
+	refs = append(refs, &config.ServingInfo.KeyFile)
+	refs = append(refs, &config.ServingInfo.ClientCA)
+	for i := range config.ServingInfo.NamedCertificates {
+		refs = append(refs, &config.ServingInfo.NamedCertificates[i].CertFile)
+		refs = append(refs, &config.ServingInfo.NamedCertificates[i].KeyFile)
+	}
+
+	for i := range config.ExtensionScripts {
```
Comment (spadgett) [inline pkg/apis/webconsole/util/helpers.go:20]: None of the extension properties are file references any longer, so we should be able to remove these.

File: pkg/apis/webconsole/validation/validation.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+			nameTaken[extConfig.Name] = true
+		}
+	}
+
+	return validationResults
+}
+
+var extNameExp = regexp.MustCompile(`^[A-Za-z0-9_]+$`)
+
+func ValidateAssetExtensionsConfig(extConfig v1.AssetExtensionsConfig, fldPath *field.Path) field.ErrorList {
```
Comment (spadgett) [inline pkg/apis/webconsole/validation/validation.go:83]: The extensions array is not being used and will be removed entirely. We could get rid of it in this PR or a follow on.

File: pkg/apis/webconsole/util/helpers.go (hunk 1 above)
Comment (deads2k) [reply pkg/apis/webconsole/util/helpers.go:20]: > None of the extension properties are file references any longer, so we should be able to remove these.

follow on

File: pkg/apis/webconsole/validation/validation.go (hunk 2 above)
Comment (deads2k) [reply pkg/apis/webconsole/validation/validation.go:83]: > The extensions array is not being used and will be removed entirely. We could get rid of it in this PR or a follow on.

follow on

**Comment by spadgett** [comment]
/close

Replaced by #16

**Pull request closed without merging**